SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -30,6 +30,13 @@ class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 	UPROPERTY(EditAnywhere, Config, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
 	TSubclassOf<UAbstractLockingPolicy> DefaultLayerLockingPolicy;
 
+	/**
+	  * Any classes not specified on another layer will be handled by the default layer, but this also gives a way
+	  * to force classes to be on the default layer.
+	  */
+	UPROPERTY(EditAnywhere, Category = "Multi-Worker")
```

Comment:
```suggestion
	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
```
This is required as all the other options are greyed out when multi-worker is disabled except this one.